public/js/components/Header/EditHeader.js

```diff
@@ -12,7 +12,7 @@ class EditHeader extends React.Component {
 
   static propTypes = {
     atom: atomPropType,
-    presence: PropTypes.object,
+    presence: PropTypes.bool,
```

Comment:
This type was wrong, which it seems prevented the presence check from being used in the code